File: JS - Fundamentals/03. Arrays - Exercise/magicSum.js
function solve(array, magicSum) {

    for (let i = 0; i < array.length; i++) {
        for (let k = i+1; k < array.length; k++) {
            if(array[i] + array[k] === magicSum){
                console.log(array[i], array[k]);
            }
        }
    }
}

solve(
    [14, 20, 60, 13, 7, 19, 8],
27


)

File: Java Fundamentals/03. Arrays - Exercise/src/MagicSum.java
import java.util.Arrays;
import java.util.Scanner;

public class MagicSum {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int[] arr = Arrays
                .stream(scan.nextLine().split(" "))
                .mapToInt(Integer::parseInt)
                .toArray();

        int sum = Integer.parseInt(scan.nextLine());

        for (int i = 0; i < arr.length; i++) {
            int current = arr[i];

            for (int j = i + 1; j < arr.length; j++) {
                if(current + arr[j] == sum){
                    System.out.println(current + " " + arr[j]);
                }
            }
        }
    }
}